docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+services:
+  backend-1:
```

Comment:
I am not familir with Docker compose, so.... is this naming convention that must be followed? Or can we use more meaningful names? (I assume former)

Reply:
Oh actually it looks like this can be freely selected (looking at later entries). So would it make sense to call it "cassandra-1" or "storage-1"?

Reply:
I assume this mostly just follows existing naming convention within Datastax tho. If so, perhaps that makes sense.

Reply:
I preserved the previous conventions from the copies in [docker images](https://github.com/stargate/docker-images), but agree this could be more descriptive.